FEXCore/Source/Interface/Context/Context.h

```diff
@@ -174,6 +174,10 @@ namespace FEXCore::Context {
       void ClearCodeCache(FEXCore::Core::InternalThreadState *Thread) override;
       void InvalidateGuestCodeRange(FEXCore::Core::InternalThreadState *Thread, uint64_t Start, uint64_t Length) override;
       void InvalidateGuestCodeRange(FEXCore::Core::InternalThreadState *Thread, uint64_t Start, uint64_t Length, CodeRangeInvalidationFn callback) override;
+      FEXCore::ForkableSharedMutex *GetCodeInvalidationMutex() override {
```

Comment:
There is no ownership transfer here and the returned pointer is unconditionally assumed to be non-null, so returning a reference here is a better fit.

Reply:
Switched over to a reference